[2020-06-15 19:36:16,866] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:79} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "uri": "/signin?email=xmush%40gmail.com&password=alta123", "request": null, "response": {"status": "UNAUTHORIZED", "message": "invalid key or secret"}}
[2020-06-15 19:36:26,522] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:79} ERROR - REQUEST_LOG	{"method": "GET", "code": "404 NOT FOUND", "uri": "/signin?email=xmush%40gmail.com&password=biasa", "request": null, "response": {"status": "UNAUTHORIZED", "message": "invalid key or secret"}}
[2020-06-15 19:37:24,097] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:69} WARNING - REQUEST_LOG	{"method": "PATCH", "code": "200 OK", "uri": "/user?", "request": null, "response": {"id": 5, "name": "Muhammad Sholeh", "phone": "081321324341", "email": "xmush@gmail.com", "user_type": "user", "is_publisher": false, "address": null, "profil_pict": null, "KTP_number": null, "KTP_pict": null, "password": "a463e8d1c38247ba9b321663f470c6fa26fa0c255cab3dae87c9ffdcfd065b62918c719ad168fa581d26397124775c62b454846117ff4517d17adbbaa4d1d05d", "created_at": "Mon, 15 Jun 2020 14:03:07 -0000", "updated_at": "Mon, 15 Jun 2020 19:37:24 -0000"}}
[2020-06-15 19:37:33,845] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:69} WARNING - REQUEST_LOG	{"method": "GET", "code": "200 OK", "uri": "/signin?email=xmush%40gmail.com&password=alta123", "request": null, "response": {"token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE1OTIyMjQ2NTMsIm5iZiI6MTU5MjIyNDY1MywianRpIjoiZjYxNzk2MmEtMDhlOC00ZDUyLTllNzAtODY3NzU2M2I5YTdiIiwiZXhwIjoxNTkyMzExMDUzLCJpZGVudGl0eSI6InhtdXNoQGdtYWlsLmNvbSIsImZyZXNoIjpmYWxzZSwidHlwZSI6ImFjY2VzcyIsInVzZXJfY2xhaW1zIjp7ImlkIjo1LCJuYW1lIjoiTXVoYW1tYWQgU2hvbGVoIiwidXNlcl90eXBlIjoidXNlciIsImlzX3B1Ymxpc2hlciI6ZmFsc2UsImlkZW50aWZpZXIiOiJhZHNfbWFya2V0cGxhY2UifX0.YUxBSMT9dMCzQ5bPB-mrBVih-woiyfv3ehESByaPjUo"}}
[2020-06-15 19:38:48,618] {/home/alta7/.local/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /user/profile [POST]
Traceback (most recent call last):
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta7/Documents/project_griffin_backend/blueprints/user/resources.py", line 164, in post
    user = Users(data['name'], data['phone'], data['email'], data['user_type'], hash_pass, salt)
TypeError: __init__() missing 4 required positional arguments: 'KTP_number', 'KTP_pict', 'password', and 'salt'
[2020-06-15 19:38:48,622] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:79} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/user/profile?", "request": {"name": "Andre Novado", "phone": "081321324351", "email": "andreh@gmail.com", "password": "alta123"}, "response": {"message": "Internal Server Error"}}
[2020-06-15 19:39:44,597] {/home/alta7/.local/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /user/profile [POST]
Traceback (most recent call last):
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta7/Documents/project_griffin_backend/blueprints/user/resources.py", line 169, in post
    user = Users(data['name'], data['phone'], data['email'], data['user_type'], hash_pass, salt)
TypeError: __init__() missing 4 required positional arguments: 'KTP_number', 'KTP_pict', 'password', and 'salt'
[2020-06-15 19:39:44,601] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:79} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/user/profile?", "request": {"name": "Andre Novado", "phone": "081321324351", "email": "andreh@gmail.com", "password": "alta123"}, "response": {"message": "Internal Server Error"}}
[2020-06-15 19:48:00,994] {/home/alta7/.local/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /user/profile [POST]
Traceback (most recent call last):
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta7/Documents/project_griffin_backend/blueprints/user/resources.py", line 165, in post
    user = Users(data['name'], data['phone'], data['email'], data['user_type'], hash_pass, salt)
TypeError: __init__() missing 4 required positional arguments: 'KTP_number', 'KTP_pict', 'password', and 'salt'
[2020-06-15 19:48:00,997] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:79} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/user/profile?", "request": {"name": "Andre Novado", "phone": "081321324351", "email": "andreh@gmail.com", "password": "alta123"}, "response": {"message": "Internal Server Error"}}
[2020-06-15 19:53:34,480] {/home/alta7/.local/lib/python3.6/site-packages/flask/app.py:1892} ERROR - Exception on /user/profile [POST]
Traceback (most recent call last):
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 468, in wrapper
    resp = resource(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask/views.py", line 89, in view
    return self.dispatch_request(*args, **kwargs)
  File "/home/alta7/.local/lib/python3.6/site-packages/flask_restful/__init__.py", line 583, in dispatch_request
    resp = meth(*args, **kwargs)
  File "/home/alta7/Documents/project_griffin_backend/blueprints/user/resources.py", line 165, in post
    user = Users(data['name'], data['phone'], data['email'], data['user_type'], hash_pass, salt)
TypeError: __init__() missing 4 required positional arguments: 'KTP_number', 'KTP_pict', 'password', and 'salt'
[2020-06-15 19:53:34,484] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:79} ERROR - REQUEST_LOG	{"method": "POST", "code": "500 INTERNAL SERVER ERROR", "uri": "/user/profile?", "request": {"name": "admin", "phone": "082326386609", "email": "admin@gmail.com", "password": "alta123"}, "response": {"message": "Internal Server Error"}}
[2020-06-15 19:54:12,290] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:69} WARNING - REQUEST_LOG	{"method": "POST", "code": "200 OK", "uri": "/user/profile?", "request": {"name": "admin", "phone": "082326386609", "email": "admin@gmail.com", "password": "alta123"}, "response": {"id": 1, "name": "admin", "phone": "082326386609", "email": "admin@gmail.com", "user_type": "user", "is_publisher": false, "address": "", "profil_pict": "", "KTP_number": "", "KTP_pict": "", "password": "b3d0cf80cb9bbe89c1c8de3d5093ae44a915cdb8c4344f4c0062b2930395127a0facf2225bdb30857d2788e42439ec62baedcccb0e1690f1023d0782d701500f", "created_at": "Mon, 15 Jun 2020 19:54:12 -0000", "updated_at": null}}
[2020-06-15 19:56:49,727] {/home/alta7/Documents/project_griffin_backend/blueprints/__init__.py:69} WARNING - REQUEST_LOG	{"method": "POST", "code": "200 OK", "uri": "/user/profile?", "request": {"name": "admin", "phone": "082326386609", "email": "admin@gmail.com", "user_type": "admin", "password": "alta123"}, "response": {"id": 1, "name": "admin", "phone": "082326386609", "email": "admin@gmail.com", "user_type": "admin", "is_publisher": false, "address": "", "profil_pict": "", "KTP_number": "", "KTP_pict": "", "password": "4ca22bfea8b4fb08216ab1a35e9e371565bf8c3f4870c96168a7ebce034dcfa74be4213eb0555e78effe60969b84ee527f5ebcf73c2c09c91edf723780a2eea3", "created_at": "Mon, 15 Jun 2020 19:56:49 -0000", "updated_at": null}}
